apps/omg_utils/lib/omg_utils/http_rpc/response.ex

```diff
@@ -16,6 +16,8 @@ defmodule OMG.Utils.HttpRPC.Response do
   Serializes the response into expected result/data format.
   """
   alias OMG.Utils.HttpRPC.Encoding
+  @version String.replace(elem(System.cmd("git", ["rev-parse", "--short=7", "HEAD"]), 0), "\n", "")
```

Comment:
names clash here - this here is the `sha`, by version I take it we understand the entire semver `v0.2.2+12a356b` string